charts/iperf3/values.yaml

```diff
@@ -4,3 +4,5 @@ service:
   # externalIp:
   # the port to bind on the service, if enabled.
   port: 40000
+# uncomment for verbose logging
+# verbose: true
```

Comment:
please just uncomment and use the defaulft, so just use `verbose: false` - no need to uncomment it, makes it harder to read what is available and what is just a comment

Reply:
Done, see latest rebase.